### PR TITLE
Feature/overview facelift

### PR DESCRIPTION
**Pull request by iimpulse**
Tons of changes in here. Definitely too many changes.

Entire page layout css needed a rework for the nodes.
Changing node banner style
Raising information in cards
Authoritative Links in external resources section
Deleting stale code / comments

### CODE REVIEW

File: src/components/ExacGeneSummary.vue [hunk 1]
```diff
@@ -139,22 +141,21 @@ export default {
...
             };
+          } else {
+            this.$emit('show-exac', false);
           }
+
         })
         .catch((err) => {
           // eslint-disable-next-line
+          this.$emit('show-exac', false);
           console.log('mygene.info error', err.message);
```
Comment (jen-martin) [inline src/components/ExacGeneSummary.vue:152]: Probably need to keep the eslint-disable comment and the console.log lines together ;-)

File: src/views/Node.vue [hunk 2]
```diff
@@ -697,32 +764,43 @@ export default {
...
+        this.authoritiveXref.gene.clinical = {
+          id: this.nodeId,
           label: 'Varsome',
           uri: `https://varsome.com/gene/${this.nodeId}`
-        });
+        };
       }
-
+      console.log(this.authoritiveXref);
+      console.log(this.references);
```
Comment (jen-martin) [inline src/views/Node.vue:803]: Probably should delete/comment out

**Review by jen-martin — COMMENTED** [review]
Looks good to me other than a few stray console.log's.

**Review by kshefchek — COMMENTED** [review]
layout changes look awesome!

replaceAll is a relatively new function could break the app for users with older browsers, we may want to use the older replace with the 'g' flag

edit: this is a .browserslistrc and babel configuration, we don't need to worry about this in the code

**Review by kshefchek — APPROVED** [review]

**Pull request merged**